cmd/posix.go

```diff
@@ -295,6 +298,9 @@ func (s *posix) Close() error {
 }
 
 func (s *posix) IsOnline() bool {
+	if atomic.LoadInt32(&s.ioErrCount) > maxAllowedIOError {
```

Comment:
With this change, we will be returning `errDiskNotFound` at https://github.com/minio/minio/pull/7972/files#diff-f8fc6e20d6f8c4f6e4e4f3c8bb79ff0eR356 instead of returning `errFaultyDisk`

Reply:
Yes, that is expected @kannappanr - we are simply indicating that disk is not online.

Reply:
Previously, would it have sent `errFaultydisk` ?

Reply:
It would incorrectly think the disk is online, the reason we want to ignore is to avoid performing I/O operations on faulty disks which cannot be recovered.

Reply:
You are also right we may simply say disk is online and let the Operation itself fail.